build_jar.py

```diff
@@ -95,4 +98,7 @@
         ),
         os.path.join(args.source, args.target),
     )
-    copyfile(args.target, os.path.join(args.source, "python", "tests", args.target))
+
+    # Copy the built jar file to the "python/tests" folder if it exists.
+    if os.path.isdir(os.path.join(args.source, "python", "tests")):
+        copyfile(args.target, os.path.join(args.source, "python", "tests", args.target))
```

Comment:
do we want to copy from line 99? 

```suggestion
        copyfile(os.path.join(args.source, args.target), os.path.join(args.source, "python", "tests", "konduit.jar"))
```